packages/ckeditor5-engine/src/view/placeholder.ts

```diff
@@ -299,6 +308,19 @@ function getChildPlaceholderHostSubstitute( parent: Element ): Element | null {
 	return null;
 }
 
+/**
+ * Displays a deprecation warning message in the console, but only once per page load.
+ */
+function showDeprecationWarning() {
```

Comment:
```suggestion
function showPlaceholderTextDeprecationWarning() {
```

Similar as for the `hasDisplayedWarning` const name.